images/linux/scripts/installers/git.sh

```diff
@@ -8,14 +8,21 @@
 source $HELPER_SCRIPTS/install.sh
 
 ## Install git
-add-apt-repository ppa:git-core/ppa -y
-apt-get update
-apt-get install git -y
+latest_git_version=$(curl -s -L "https://api.github.com/repos/git/git/tags" | jq -r '.[].name' | grep -ve "-.*" | head -1 | tr -d "v")
+download_with_retries https://github.com/git/git/archive/v${latest_git_version}.tar.gz "/tmp" "git.tar.gz"
+git_installation_path="/usr/local/git"
+git_tar_tmp="/tmp/git.tar.gz"
+mkdir -p "${git_installation_path}"
+tar -zxf "${git_tar_tmp}" -C "${git_installation_path}"
+rm -f "${git_tar_tmp}"
+cd "${git_installation_path}/git-${latest_git_version}"
+make prefix=/usr/local all
+make prefix=/usr/local install
```

Comment:
Do we really need to build git from source code during image-generation? I don't think that it is good solution.
I think both `git` and `git-lfs` should be available in default apt sources without adding additional repos and installation will be faster

Reply:
@maxim-lobanov thank you, simplified installation